### DISCUSSION TITLE
How do you "remove" the config for a directory?

### DISCUSSION THREAD

**Discussion by jvalrog** [jdx/mise#3423] [General]
Was trying the tool, set node@22 on a directory and a global version node@20.
Now I want to remove that node@22 for that directory, but can't find how.
Documentation is not clear or is missing.

**Comment by jvalrog** [comment]
Close because apparently I had to remove the config file from the directory

**Reply by syhol** [reply]
There is a feature request open for this: https://github.com/jdx/mise/issues/1465

**Reply by Marukome0743** [reply]
Following up on this older discussion: removing the entire config file is no longer necessary.

`mise unuse` was added in [#3627](https://github.com/jdx/mise/pull/3627) and released in mise v2024.12.15. To remove the local Node selection and fall back to the global configuration, run this from the project directory:

```sh
mise unuse node
```

By default, mise will also prune the removed version if no other config uses it. To remove only the config entry while keeping the installed runtime, use:

```sh
mise unuse --no-prune node
```

The global `node@20` configuration remains untouched, so it becomes active again after the local entry is removed.

I verified the current behavior on main, including local, global, environment-specific, and multi-version configurations.

*AI-assisted — Tool: Codex; model: OpenAI/GPT-5; version: unavailable.*
